docs/api/ethereum.md

```diff
@@ -1089,6 +1115,30 @@ const assetId = '100';
 const accountId = await ledger.getAssetAccount(assetId);
 ```
 
+### getAssetIdAt(index)
+
+An `asynchronous` class instance `function` which returns the asset id at specified `index`.
+
+**Arguments:**
+
+| Argument | Description
+|-|-
+| index | [required] A `number` representing the asset index.
+
+**Result:**
+
+A `number` representing the asset id.
+
+**Example:**
+
+```ts
+// arbitrary data
+const index = '100';
```

Comment:
Same thing as above. Add comment that 300 tokens exists so indexes from 0 to 299 are available or samething like that.